app/channels/iteration_channel.rb

```diff
@@ -1,7 +1,9 @@
 class IterationChannel < ApplicationCable::Channel
   def subscribed
     # Assert that the user owns this iteration
-    iteration = current_user.iterations.find_by!(uuid: params[:uuid])
+    iteration = Iteration.find_by!(uuid: params[:uuid])
+
+    return unless iteration.viewable_by?(current_user)
```

Comment:
Is there any reasonable way in which a non-owner subscribes to this channel? In other words, should we somehow log this as an error?

Reply:
If it's not expected, an error would be logical, yes.

Reply:
@kntsoriano can you add a TODO to raise a bugsnag in this scenario pls.